tests/parsers/appbio_quantstudio/appbio_quantstudio_structure_test.py

```diff
@@ -142,7 +142,7 @@ def test_header_builder_invalid_plate_well_count() -> None:
 @pytest.mark.short
 def test_header_builder_no_header_then_raise() -> None:
     header_contents = get_raw_header_contents(raw_text="")
-    lines = _read_to_lines(header_contents)
+    lines = _read_to_lines(header_contents, encoding="UTF-8")
```

Comment:
without these changes, we get `allotropy.exceptions.AllotropeConversionError: Unable to detect text encoding for file. The file may be empty.`